secretsstoresynccontroller/config/samples/azurespcmultiplesecrets.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: spc-multiple-secrets                        # name of the SecretProviderClass
+spec:
+  # Secret synchronization fields removed from SPC and moved to SecretStoreySyncObject
+  # secretObjects:                              # [OPTIONAL] SecretObject defines the desired state of synced K8s secret objects
+  # - data:
+  #  - key: username                           # data field to populate
+  #    objectName: foo1                        # name of the mounted content to sync. this could be the object name or the object alias
+  #  secretName: foosecret                     # name of the Kubernetes Secret object
+  #  type: Opaque                              # type of the Kubernetes Secret object e.g. Opaque, kubernetes.io/tls
+  # ...
+  provider: azure                                   # name of the provider
+  parameters:
+    clientID: "<client-id-for-workload-identity>"   # [OPTIONAL for Azure] used for azure authentication
+    keyvaultName: "<key-vault-name>"                # name of the keyvault
+    objects: |                                      # [REQUIRED] objects to fetch from keyvault, specified in an
+      array:                                        # array of objects.
+        - |
+          objectName: <secret1-name-in-key-vault>   # name of the secret object
+          objectType: secret                        # object types: secret, key or cert
+        - |
+          objectName: <secret2-name-in-key-vault>   # name of the secret object
+          objectType: secret                        # object types: secret, key or cert
+          objectAlias: <secret2-object-alias>       # [OPTIONAL] object alias, default to object name
+          objectVersion:                            # [OPTIONAL] object versions, default to latest if empty
+    tenantId: "<tenant-id>"                         # [OPTIONAL for Azure] used for azure authentication
```

Comment:
Instead of an azure specific example, add an example with the e2e provider

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/6d96251b16078612abfc267dd68de090493113ca/test/bats/tests/e2e_provider/e2e_provider_v1_secretproviderclass_ns.yaml#L37-L45